### ISSUE TITLE
Windows VST3s load in Audacity, but UIs are non-functional.

### ISSUE THREAD

**Issue by Lowclock** [lsp-plugins/lsp-plugins#385]
GNU Make 4.4
MinGW-W64 8.1.0
PHP 8.3.3
Audacity 3.4.2
LSP 1.2.15
Windows 10 Pro, Ryzen 3700x

VST3 Plugins are discovered and enabled and load correctly, but their UIs don't seem to actually do anything. Controls can be manipulated, but things such as curves in the FFT in the Filter Plugin are not present. Hitting Preview doesn't generate anything in the FFT window or VU meters. Hitting apply sometimes seems to work as if it the plugin was in its default configuration, but sometimes do nothing at all. File loading, such as in Impulse Response/Reverb, is not reflected in the UI, or in it's results after hitting apply. There is nothing in Audacity's log that suggests there is a problem, but it's clearly not working as it should.

**Comment by sadko4u** [comment]
Have you tried another plugins before posting the issue?
AFAIK Audacity is still poorly supporting plugin formats, especially VST3 and LV2.

**Comment by Lowclock** [comment]
Yes. I have tons of other plugins that work correctly in Audacity, The only ones I can even think of that don't work 100% correctly are the Fabfilter series, which just don't have functional spectrum analyzers and VU meters. Everything else including stuff from Izotope, Waves, Reaper, Brainworx, and much more work perfectly.

**Comment by sadko4u** [comment]
@Lowclock how do you enable Audacity to see VST3 plugins for Windows?

**Comment by sadko4u** [comment]
Could not reproduce as the way of reproducing the problem was not reported.
